etc/kayobe/kolla.yml

```diff
@@ -139,10 +135,6 @@ kolla_sources:
     type: git
     location: https://github.com/stackhpc/magnum.git
     reference: stackhpc/{{ openstack_release }}
-  neutron-base:
-    type: git
-    location: https://github.com/stackhpc/neutron.git
-    reference: stackhpc/{{ openstack_release }}
   neutron-base-plugin-networking-generic-switch:
     type: git
     location: https://github.com/stackhpc/networking-generic-switch.git
```

Comment:
These are required (in `kolla_sources`) to avoid going backwards:
```
  prometheus-memcached-exporter:
    version: 0.14.4
    sha256:
      amd64: 60b0fb90e55937869e2a7aa1310d148d56a28639ca3373e0c50548ff8eebaaa1
      arm64: 053c6020bb2321d301bb1d43fbfc5aec64c3104a45a94338b4ca9351fc9ee014
  prometheus-v2-server:
    version: 2.53.1
    sha256:
      amd64: 2234aa0f66d9f9b854144f6faaaed72a316df7a680d9dad7cb48e49a6fa5332c
      arm64: a7f28c83c3c943b953a9d00860bd3f2422464fb7c27a3c4037ef1ce2a41592b5
```

Reply:
Didn't work :(